src/sync/installation.ts

```diff
@@ -418,6 +435,11 @@ export const processInstallation =
 		return async (data: BackfillMessagePayload, sentry: Hub, logger: LoggerWithTarget): Promise<void> => {
 			const { installationId, jiraHost } = data;
 
+			// This is prefered at taskType definition, but feature flag requires async so was moved to starting of process
+			if (!await booleanFlag(BooleanFlags.BACKFILL_FOR_BUILDS_AND_DEPLOYMENTS, true, jiraHost)) {
+				taskTypes = taskTypes.filter((task) => task !== "build" && task !== "deployment")
+			}
+
```

Comment:
Feature flag added at this point, ideally wanted it up top when taskTypes is defined but need async for flag,
this was the least intrusive way.